reference/build_helpers/cmake.rst

```diff
@@ -237,3 +237,43 @@ Installs `CMake` project with the given parameters.
 Parameters:
     - **args** (Optional, Defaulted to ``None``): A list of additional arguments to be passed to the ``cmake`` command. Each argument will be escaped according to the current shell. No extra arguments will be added if ``args=None``.
     - **build_dir** (Optional, Defaulted to ``None``): CMake's output directory. If ``None`` is specified the ``build_folder`` from ``configure()`` will be used.
+
+
+patch_config_paths()
+++++++++++++++++++++
+
```

Comment:
Maybe mark as "experimental"? I see potential point of evolution

Reply:
ok!